packages/core/upload/admin/src/components/FolderCard/FolderCardBodyAction/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+import styled from 'styled-components';
+
+const FolderCardLink = styled(NavLink)`
+  max-width: 100%;
+  text-decoration: none;
```

Comment:
Do these styles need to be applied to the `button` too? Wondering, because the title can be truncated by CSS (using `ellipsis` on `Typography`).